xml_inscriptions/transcribed/MA.Bos.MFA.G.01.Res.08.34b.xml

```diff
@@ -113,49 +106,51 @@
                   <biblScope unit="item">no. 231</biblScope>
                </bibl>
                <bibl>
-                  <ref type="external" target="xx">Title of Reference</ref>
+                  <ref type="external"
+                     target="https://scholarlyeditions-brill-com.revproxy.brown.edu/reader/urn:cts:greekDoc:seg.028-0231.sego:lem-ap/"
```

Comment:
the one issues with the link you've given here is that it will attempt to authenticate against Brown credentials; give the static url, not the one through Brown's revproxy.